commit 71b49e299dbe22853095119da5064303e1d6b9ff
Author: Florian Weimer <fweimer@redhat.com>
Date:   Tue Jan 21 10:36:58 2025 +0100

    nptl: Include <stdbool.h> in tst-skeleton-affinity-inheritance.c
    
    The file uses the identifiers bool, false, true.

diff --git a/nptl/tst-skeleton-affinity-inheritance.c b/nptl/tst-skeleton-affinity-inheritance.c
index 926f49622990e9e4..e1f328ae265b2bfb 100644
--- a/nptl/tst-skeleton-affinity-inheritance.c
+++ b/nptl/tst-skeleton-affinity-inheritance.c
@@ -32,6 +32,7 @@
    they're both inherited.  */
 
 #include <errno.h>
+#include <stdbool.h>
 #include <stdio.h>
 #include <support/test-driver.h>
 #include <support/xthread.h>
